src/OpenTelemetry.Instrumentation.ServiceFabricRemoting/OpenTelemetry.Instrumentation.ServiceFabricRemoting.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Description>ServiceFabric Remoting instrumentation for OpenTelemetry .NET.</Description>
+    <PackageTags>$(PackageTags);distributed-tracing</PackageTags>
+    <MinVerTagPrefix>Instrumentation.ServiceFabricRemoting-</MinVerTagPrefix>
+    <PackageValidationBaselineVersion>1.9.0</PackageValidationBaselineVersion>
+    <AnalysisLevel>latest-all</AnalysisLevel>
```

Comment:
Analysis level is enabled for whole solution. You should remove this.

Reply:
fixed